build.gradle

```diff
@@ -1,6 +1,14 @@
 group 'multi-int-store'
 version '0.1.0-SNAPSHOT'
 
-repositories {
-    mavenCentral()
-}
\ No newline at end of file
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath 'org.owasp:dependency-check-gradle:4.0.1'
+    }
+}
+
+apply plugin: 'org.owasp.dependencycheck'
```

Comment:
Where do you set the threshold for what threat level we want to allow?

Reply:
Updated with the configuration section